experiments.json

```diff
@@ -16,5 +16,13 @@
       "min": "0",
       "max": "100"
     }
+  },
+  "whatsnew-notification": {
+    "match": {
+    },
```

Comment:
Nit: if match doesn't have any parameters, just have the open/close {} on the same line.